pkg/util/ghactions/runner.go

```diff
@@ -42,43 +54,23 @@ func InitGHRunnerArgs(token, name, repoURL string) error {
 }
 
 func GetToken() string {
-	if (args == &RunnerArgs{}) {
-		return ""
-	}
-	return args.Token
+	return util.If(args != nil,
+		args.Token,
+		"")
```

Comment:
maybe this also needs to be using `if (args != nil)` ?

Reply:
yeah will update shortly, thx for catching it

Reply:
done

Reply:
well..actually created a new ternary op supporting nillable cases